packages/wonder-blocks-tooltip/src/util/ref-tracker.ts

```diff
@@ -41,8 +39,7 @@ export default class RefTracker {
                 throw new Error("targetFn must be a function");
             }
 
-            // @ts-expect-error [FEI-5019] - TS2322 - Type '((instance: any) => void) | null' is not assignable to type '(arg1?: HTMLElement | null | undefined) => void | null | undefined'.
-            this._targetFn = targetFn || null;
+            this._targetFn = targetFn || undefined;
```

Comment:
I feel like TS' type system tends to favour `undefined` over `null` for some reason.

Reply:
It's not that. It's that because I don't initialize `_targetFn`, it is defaulted to `undefined`. And I didn't bother keeping the ability to type it to `null`, so `null` wasn't valid here.